geth/transactions/txqueue_manager_test.go

```diff
@@ -197,10 +197,9 @@ func (s *TxQueueTestSuite) TestAccountMismatch() {
 		To:   common.ToAddress(TestConfig.Account2.Address),
 	})
 
-	err := txQueueManager.QueueTransaction(tx)
-	s.NoError(err)
+	txQueueManager.QueueTransaction(tx)
```

Comment:
In some places, the error is checked
```
s.NoError(txQueueManager.QueueTransaction(tx))
```
and in here it's not. It should be consistent. Btw. why does linter not complain that error is not checked?

Reply:
maybe linter doesn't complain in tests

Reply:
You're right. We would need to add `"Test": true` to `.gometalinter.json` to have test files checked.